prog/mutation.go

```diff
@@ -501,7 +501,11 @@ func (t *IntType) getMutationPrio(target *Target, arg Arg, ignoreSpecial bool) (
 		return plainPrio, false
 	}
 
-	switch size := t.RangeEnd - t.RangeBegin + 1; {
+	size := t.RangeEnd - t.RangeBegin + 1
+	if t.Align != 0 {
+		size = (t.RangeEnd-t.RangeBegin)/t.Align + 1
```

Comment:
For the [0:-1] special case we will into the plainPrio below, right? I think that's right behavior.

Reply:
Actually, that's an oversight. Do we really want to go into plainPrio if we have e.g., `int8[0:-1, 50]`? That's less than 6 values, so I'd have thought we wanted to go into `size <= 15`.